datalad_registry/conf.py

```diff
@@ -36,6 +36,9 @@ class BaseConfig(OperationConfig):
         "metalad_studyminimeta",
         "datacite_gin",
         "bids_dataset",
+        # === DANDI related extractors ===
+        "dandi:dandiset",
+        # "dandi:assets",  # Let's not activate this yet by default
```

Comment:
Let's go (here or right after) for agreeing on "semantic" that without `:SMTH` it is dataset level metadata, and then with `:files` it would be the list with per-file metadata.

```suggestion
        "dandi",
        # "dandi:files",  # Let's not activate this yet by default
```

`files` since it is per "file" metadata (although .zarr would be a folder) and "asset" is an internal DANDI term, as well as "dandiset".